Cargo.toml

```diff
@@ -15,9 +15,13 @@ schemars = "0.8.1"
 cosmwasm-schema = "1.1.4"
 cw-utils = "0.13"
 anyhow = "1"
+elys-bindings = { version = "0.5.0", git = "https://github.com/elys-network/bindings", branch = "feat/import-financial" }
```

Comment:
Need to use `tag` and not `branch` to declare dependencies.

Reply:
@politeWall yep it will be changed to be tag once approved.